packages/selinux-policy/files.cil

```diff
@@ -9,6 +9,7 @@
 (classmapping files relabel relabel_blk_file)
 (classmapping files relabel relabel_sock_file)
 (classmapping files relabel relabel_fifo_file)
+(classmapping files relabel relabel_anon_inode)
```

Comment:
This treatment of `anon_inode` looks OK.

In the absence of the transition rules, it looks like the `anon_inode` objects will continue to use the subject's label, and be covered by this blanket rule:
```
(allow all_s self (files (mutate)))
```

To actually write policy for this, we'd need a named transition rule and additional rules like:
```
(typetransition all_s all_s anon_inode "[userfaultfd]" userfaultfd_t)
(allow trusted_s userfaultfd_t (files (mutate)))
(neverallow untrusted_s userfaultfd_t (files (mutate)))
```

In other words - force all userfaultfd descriptors to be created with a particular label, and then restricting mutations to just the subjects that we want to be able to use them.

Before we did that, we'd probably want to default to turning off userfaultfd for unprivileged users via `vm.unprivileged_userfaultfd = 0`, and then understand why that wasn't sufficient.